code/game/sound.dm

```diff
@@ -208,35 +208,35 @@ distance_multiplier - Can be used to multiply the distance at which the sound is
 /proc/get_sfx(soundin)
 	if(istext(soundin))
 		switch(soundin)
-			if (SFX_SHATTER)
+			if(SFX_SHATTER)
 				soundin = pick('sound/effects/glassbr1.ogg','sound/effects/glassbr2.ogg','sound/effects/glassbr3.ogg')
-			if (SFX_EXPLOSION_CREAKING)
+			if(SFX_EXPLOSION)
 				soundin = pick('sound/effects/explosion1.ogg','sound/effects/explosion2.ogg')
-			if (SFX_EXPLOSION_CREAKING)
+			if(SFX_EXPLOSION_CREAKING)
 				soundin = pick('sound/effects/explosioncreak1.ogg', 'sound/effects/explosioncreak2.ogg')
-			if (SFX_HULL_CREAKING)
+			if(SFX_HULL_CREAKING)
 				soundin = pick('sound/effects/creak1.ogg', 'sound/effects/creak2.ogg', 'sound/effects/creak3.ogg')
-			if (SFX_SPARKS)
+			if(SFX_SPARKS)
 				soundin = pick('sound/effects/sparks1.ogg','sound/effects/sparks2.ogg','sound/effects/sparks3.ogg','sound/effects/sparks4.ogg')
-			if (SFX_RUSTLE)
+			if(SFX_RUSTLE)
 				soundin = pick('sound/effects/rustle1.ogg','sound/effects/rustle2.ogg','sound/effects/rustle3.ogg','sound/effects/rustle4.ogg','sound/effects/rustle5.ogg')
-			if (SFX_BODYFALL)
+			if(SFX_BODYFALL)
 				soundin = pick('sound/effects/bodyfall1.ogg','sound/effects/bodyfall2.ogg','sound/effects/bodyfall3.ogg','sound/effects/bodyfall4.ogg')
-			if (SFX_PUNCH)
+			if(SFX_PUNCH)
 				soundin = pick('sound/weapons/punch1.ogg','sound/weapons/punch2.ogg','sound/weapons/punch3.ogg','sound/weapons/punch4.ogg')
-			if (SFX_CLOWN_STEP)
+			if(SFX_CLOWN_STEP)
 				soundin = pick('sound/effects/clownstep1.ogg','sound/effects/clownstep2.ogg')
-			if (SFX_SUIT_STEP)
+			if(SFX_SUIT_STEP)
 				soundin = pick('sound/effects/suitstep1.ogg','sound/effects/suitstep2.ogg')
-			if (SFX_SWING_HIT)
+			if(SFX_SWING_HIT)
 				soundin = pick('sound/weapons/genhit1.ogg', 'sound/weapons/genhit2.ogg', 'sound/weapons/genhit3.ogg')
-			if (SFX_HISS)
+			if(SFX_HISS)
 				soundin = pick('sound/voice/hiss1.ogg','sound/voice/hiss2.ogg','sound/voice/hiss3.ogg','sound/voice/hiss4.ogg')
-			if (SFX_PAGE_TURN)
+			if(SFX_PAGE_TURN)
 				soundin = pick('sound/effects/pageturn1.ogg', 'sound/effects/pageturn2.ogg','sound/effects/pageturn3.ogg')
-			if (SFX_RICOCHET)
+			if(SFX_RICOCHET)
 				soundin = pick( 'sound/weapons/effects/ric1.ogg', 'sound/weapons/effects/ric2.ogg','sound/weapons/effects/ric3.ogg','sound/weapons/effects/ric4.ogg','sound/weapons/effects/ric5.ogg')
-			if (SFX_TERMINAL_TYPE)
+			if(SFX_TERMINAL_TYPE)
 				soundin = pick('sound/machines/terminal_button01.ogg', 'sound/machines/terminal_button02.ogg', 'sound/machines/terminal_button03.ogg', \
```

Comment:
Fix aside, `if ()` and `if()` are style choices we intentionally don't make. If you're adding new code you're encouraged to follow the pattern of what you're adding to, otherwise you should leave them be
https://github.com/tgstation/tgstation/blob/master/.github/guides/STYLE.md#things-that-do-not-matter

Reply:
i justify this by saying that the rest of the file uses no space, with only one or two exceptions, which i personally find weird, anyway. maybe it _is_ a peeve thing, though.